docs/user-guide.md

```diff
@@ -342,19 +342,21 @@ Print debugging information to `stderr`.
 ##### `-h, --help`
 Print a help message, then exit.
 
-<!-- ### `fossa report`
+### `fossa report`
 
 Print the project's license report.
 
 #### Flags
-##### `-t, --type report_type`
-Print a specific type of license report for automatically creating attribution files. Possible report types include:
-- `NOTICE`: generate a `NOTICE` file for your dependencies
-- `ATTRIBUTION`: generate an `ATTRIBUTION` file for your dependencies
-
-##### `-j, --json`
+<!-- ##### `-j, --json`
```

Comment:
Why remove report type?

Reply:
I was originally thinking of report types in the context of licenses i.e. generating attribution vs. notice files. I couldn't figure out a good way to automatically generate different types of license reports, so I removed this.

This is going to be added back in to differentiate between e.g. dependency vs. license reports.

Reply:
See above.